dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -558,7 +558,9 @@ CWWKD1055.unsupported.entity.attr=CWWKD1055E: The {0} attribute of the {1} entit
  has an unsupported type: {2}. Jakarta Data supports Java enumeration types, \
  the {3} temporal types, and the following basic types: {4}. The built-in \
  Jakarta Data provider also supports some Jakarta Persistence types, such as \
- embeddables.
+ embeddables. Alternatively, if you are using a Jakarta Persistence entity, \
+ you can add the {5} annotation to the {0} attribute to provide an \
+ AttributeConverter that converts an unsupported type to a supported type.
```

Comment:
Recommend adding a noun after AttributeConverter that indicates what the AttributeConverter is. In the CWWKD1111 message you used "implementation". If there is a better word than "implementation" (e.g., "method", "object", "feature", etc.) you can use it instead.

Reply:
Adding the word implementation here as suggested. That is the most precise noun I can think of for it.